plugins/modules/cloud/scaleway/scaleway_volume.py

```diff
@@ -51,10 +51,14 @@
     description:
      - Name used to identify the volume.
     required: true
+  project:
+    type: str
+    description:
+     - ScaleWay project ID to which volume belongs.
   organization:
     type: str
     description:
-     - ScaleWay organization ID to which volume belongs.
+     - DEPRECATED ScaleWay organization ID to which volume belongs.
```

Comment:
```suggestion
     - ScaleWay organization ID to which volume belongs.
```
We can deprecate it in a follow-up release, but please not in the release where this option is added. Also this is not the way we deprecate options.